subprojects/flashrom.wrap

```diff
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = flashrom
-url = https://github.com/hughsie/flashrom.git
+url = https://github.com/fwupd/flashrom.git
```

Comment:
Oh this fork didn't come over.  Are you planning to bring it too?

Reply:
I can do now; I think most of this is upstream now.